### ISSUE TITLE
load order issue with matb33:collection-hooks

### ISSUE THREAD

**Issue by Kestanous** [gterrono/houston#270]
I have no idea what is causing this but if I have my .meteor/packages file like this:
```
houston:admin
matb33:collection-hooks
```
I get an issue with collection-hooks:
```
TypeError: Cannot call method 'insert' of undefined
```
But it works fine like this:
```
matb33:collection-hooks
houston:admin
```

**Comment by AlexeyMK** [comment]
If I had to guess, I'd blame https://github.com/gterrono/houston/commit/64a267beba7d744420fae3663cff272f38247bc7 - @matteodem - mongo-collection-instances probably has a bug somewhere in that it doesn't copy the Collection's abilities over perfeclty. I think there was also an earlier issue about this.

@Kestanous, thank you for all of the feedback, man. @gterrono, @matteodem and I get pretty swamped with stuff, but we're always grateful for PRs!

**Comment by Kestanous** [comment]
Looks like your PR already exists: https://github.com/gterrono/houston/pull/271. Thanks @dburles!

**Comment by matteodem** [comment]
awesome!

**Comment by Kestanous** [comment]
Fixed with #271

**Comment by PranayShah** [comment]
I was still running into this issue with Houston version 2.0.6 and a package called files-collection version 1.2.0. Having files-collection below Houston in packages.js solved it for me as well.
